### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by zalfenior**
Documenting Creditor, ShopScript, StateManager, ValidLoan and PlayerHealth

### CODE REVIEW

File: fiscal-shock/Assets/Scripts/Finance/ShopScript.cs [hunk 1]
```diff
@@ -62,6 +78,10 @@ public class ShopScript : MonoBehaviour {
         }
     }
 
+    /// <summary>
+    /// Sets up the players inventory and shotting script for later use.
```
Comment (pagewin) [inline fiscal-shock/Assets/Scripts/Finance/ShopScript.cs:82]: *shooting
*shop's

File: fiscal-shock/Assets/Scripts/Player/PlayerHealth.cs [hunk 2]
```diff
@@ -58,6 +66,9 @@ public class PlayerHealth : MonoBehaviour {
         }
     }
 
+    /// <summary>
+    /// Shows the player flashlight within the dungeons.
```
Comment (pagewin) [inline fiscal-shock/Assets/Scripts/Player/PlayerHealth.cs:70]: Wrong actually -- this enables invincibility when the player loads. The flashlight part is to change the color so the player has some visual feedback that something is different

File: fiscal-shock/Assets/Scripts/Player/PlayerHealth.cs [hunk 3]
```diff
@@ -71,6 +82,10 @@ public class PlayerHealth : MonoBehaviour {
         yield return null;
     }
 
+    /// <summary>
+    /// Used to disable invincibility after players have a few seconds to position
+    /// or if they are currently attacking themselves.
```
Comment (pagewin) [inline fiscal-shock/Assets/Scripts/Player/PlayerHealth.cs:87]: I know the grammar is technically correct but it sounds like the player is hitting themselves with this phrasing

**Comment by zalfenior** [comment]
Should be good now :)

**Pull request merged**